packages/grid/src/vaadin-grid-selection-column.js

```diff
@@ -278,6 +278,22 @@ class GridSelectionColumn extends GridColumn {
   __onDataProviderChanged() {
     this.__selectAllHidden = !Array.isArray(this._grid.items);
   }
+
+  /**
+   * Assuming the grid uses an items array data provider, fetches all the filtered items
+   * from the data provider and invokes the callback with the resulting array.
+   *
+   * @private
+   **/
```

Comment:
nit: fix the JSDoc comment style

```suggestion
   */
```